src/Forwarder.cpp

```diff
@@ -192,12 +197,6 @@ void Forwarder::forward_epics_to_kafka() {
     auto TimeAfterIterationExecution = STEADY_CLOCK::now();
     IterationExecutionDuration = std::chrono::duration_cast<MILLISECONDS>(
         TimeAfterIterationExecution - TimeAtStartOfLoop);
-    if (TimeAfterIterationExecution - TimeSinceLastStatus > 3000ms) {
-      if (status_producer_topic) {
-        report_status();
-      }
-      TimeSinceLastStatus = TimeAfterIterationExecution;
-    }
     if (IterationExecutionDuration.load() >= Dt) {
       Logger->error("slow main loop: {}",
                     IterationExecutionDuration.load().count());
```

Comment:
```suggestion
                    IterationExecutionDuration.count());
```

Reply:
Forwarder main loop is being cleaned up as part of DM-1755
(All of this code will be deleted)